cmd/branchingconfigmanagers/release-jobs-config-manager/internal/bumper/bumper.go

```diff
@@ -0,0 +1,127 @@
+package bumper
```

Comment:
There is no need for the whole package. Please check: https://github.com/openshift/ci-tools/blob/master/pkg/config/load.go#L242